cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/StandaloneAppFabricServer.java

```diff
@@ -70,11 +70,12 @@ public StandaloneAppFabricServer(CConfiguration cConf,
                                    SystemArtifactLoader systemArtifactLoader,
                                    PluginService pluginService,
                                    PrivilegesFetcherProxyService privilegesFetcherProxyService,
+                                   AppVersionUpgradeService appVersionUpgradeService,
                                    RouteStore routeStore) {
     super(cConf, sConf, discoveryService, schedulerService, notificationService, hostname, handlers,
           metricsCollectionService, programRuntimeService, applicationLifecycleService,
           programLifecycleService, streamCoordinatorClient, servicesNames, handlerHookNames, namespaceAdmin,
-          systemArtifactLoader, pluginService, privilegesFetcherProxyService, routeStore);
+          systemArtifactLoader, pluginService, privilegesFetcherProxyService, appVersionUpgradeService, routeStore);
```

Comment:
`StandaloneAppFabricServer` is used only in standalone mode, do we need `appVersionUpgradeService` here?